src/main/java/org/opentripplanner/analyst/batch/CSVPopulation.java

```diff
@@ -64,7 +64,7 @@ else if (CRS.getAxisOrder(destCrs) == CRS.AxisOrder.EAST_NORTH)
                 if (!destCrs.equals(sourceCrs)) {
                     transform = true;
 
-                    // find the transformation, being strict about datums &c.
```

Comment:
Personally I like ampersand-c. as a way to write "et cetera", as the ampersand is a ligature of Latin "et". It's fine to remove it though if it might confuse some people.

Reply:
Oh I see, Javadoc is interpreting this as in HTML. Now I see why it's removed.

Reply:
Correct, Java 11 is way more strict with these HTML entities.

I big problem where also the < and > characters. I got a little creative and converted to unicode characters where possible. If not I just spelled them out with words.

Reply:
In my commits on the 2.x port, I used `-&gt;` to replace `->`, but although I managed to eliminate all the error messages, the most complete "solution" was to just turn off the doclint in the config in the POM.